client/signup/steps/import-preview/style.scss

```diff
@@ -0,0 +1,69 @@
+.import-preview__card {
+	max-width: 360px;
+	padding: rem( 16px );
```

Comment:
I believe most of the steps have a `border-radius` of `3px`. Can we add that here for consistency?

Reply:
Thanks Jon, now added.